src/java.desktop/share/classes/java/awt/image/MultiPixelPackedSampleModel.java

```diff
@@ -638,6 +647,9 @@ public void setDataElements(int x, int y, Object obj, DataBuffer data) {
      * @param iArray the input pixel in an {@code int} array
      * @param data the {@code DataBuffer} containing the image data
      * @see #getPixel(int, int, int[], DataBuffer)
+     * @throws NullPointerException if {@code data} is null.
```

Comment:
here NPE can be thrown if iArray is null...maybe we should mention that in spec too..

Reply:
Looks like this is the one case that missed saying that.